src/controllers/confirm.company.controller.ts

```diff
@@ -27,12 +32,25 @@ const INELIGIBLE_PAGES = {
  * @param next
  */
 
-export const get = (req: Request, res: Response, next: NextFunction) => {
-  if (req.session) {
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  const session: Session = req.session as Session;
+
+  if (session) {
     try {
-      const company: ObjectionCompanyProfile = retrieveCompanyProfileFromObjectionSession(req.session);
+      const company: ObjectionCompanyProfile = retrieveCompanyProfileFromObjectionSession(session);
+
+      const token: string = retrieveAccessTokenFromSession(session);
+
+      let latestGaz1Date: string;
+      if (await getCompanyEligibility(company.companyNumber, token)) {
+        latestGaz1Date = await getLatestGaz1Date(company.companyNumber, token);
+      } else {
+        latestGaz1Date = "No notice in The Gazette";
```

Comment:
i couldn't see a test to cover the 'else' line 48 scenario

Reply:
i can see it now, it's been added to an existing one

Reply:
Yes.